google_compute_engine/ip_forwarding/tests/ip_forwarding_utils_test.py

```diff
@@ -20,25 +20,36 @@
 from google_compute_engine.test_compat import unittest
 
 
+def _create_mock_process(returncode, stdout, stderr):
```

Comment:
Thanks for updating the function! Please add a doc string for the function
(e.x.: https://github.com/GoogleCloudPlatform/compute-image-packages/blob/master/google_compute_engine/tests/config_manager_test.py).

Also, for style consistency, please rename the function to _CreateMockProcess. We haven't yet moved the code to pep8 compliant: https://github.com/GoogleCloudPlatform/compute-image-packages/issues/274